content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
 
-Open your sales journal. Under the *Advanced Settings* tab, in the **Electronic invoicing** field,
-check *FatturaPA* and *Save*.
+.. note::
+   :guilabel:`Codice Destinatario` of fake destinations: `0803HR0` - `N8MIMM9` - `X9XX79Z`.
 
-.. image:: italy/journal.png
+.. image:: italy/italy-edi.png
    :align: center
-   :alt: FatturaPA feature in the sales journal's settings
+   :alt: Italy's electronic document invoicing options
+
+Taxes
+=====
+
+Specific configurations are required for the **reverse charge** type of taxes. In case of a
+**reverse charge** tax, the seller does *not* charge the customer for the VAT but instead the
+customer pays the VAT *themselves* to their government. There are **two** main types of **reverse
+charges**: :ref:`external <italy/external-reverse>` and :ref:`internal <italy/internal-reverse>`
+reverse charge.
 
-Issue invoices
---------------
+.. _italy/external-reverse:
 
-Send an invoice the way you would do it normally. The EDI process proceeds automatically.
+External reverse charge
+-----------------------
 
-.. image:: italy/sending.png
+Invoices
+~~~~~~~~
+
+To use an **external reverse** charge tax on an invoice, first go to :menuselection:`Accounting -->
+Configuration --> Taxes` and select a `O%` tax of type :guilabel:`Sales` or :guilabel:`Purchases`,
+according to your needs. Once within the tax menu, click :guilabel:`Edit`, and at the bottom of the
+menu, tick :guilabel:`Has exoneration of tax (Italy)`, select the :guilabel:`Exoneration` that
+corresponds to your transaction, and enter the corresponding :guilabel:`Law Reference`. When done,
+click :guilabel:`Save`.
+
+.. note::
+   If you need to use a different :guilabel:`Exonerations` nature, click :menuselection:`Action -->
+   Duplicate` within the tax menu to create a copy of an existing similar tax. Then, select another
+   :guilabel:`Exoneration`, and :guilabel:`Save`. Repeat this process as many times as you need
+   different :guilabel:`Exoneration` nature taxes.
+
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
+
+.. image:: italy/italy-tax.png
    :align: center
-   :alt: Asynchronous sending on a customer invoice
+   :alt: External reverse charge settings
+
+On your invoice, select the corresponding tax you need in the :guilabel:`Taxes` field. You can find
+and edit additional info by opening the **XML** file of the issued invoice:
+
+- :guilabel:`SdI address (Codice Destinatario)`: must be filled for both **EU** or
+  **non-EU**;
+- :guilabel:`Country Id`: must contain the country of the foreign seller in the two-letter ISO
+  (Alpha-2) code (ex., `IT` for 'Italy');
+- :guilabel:`CAP`: must be filled with `00000`;
+- :guilabel:`Partita Iva` (**VAT number**): must contain the **VAT** number for **EU businesses**
+  and `OO99999999999` (double 'O' letter, not 'zero') for **non-EU businesses**. In case of private
```

Comment:
```suggestion
  and `OO99999999999` (double 'O' letter, not 'zero') for **non-EU businesses**. In case of private
```
Is it two 'O's or two '0's for EU businesses ? You put two 'O's in your task I believe